SpecificModConfigMenu.cs

```diff
@@ -469,11 +471,20 @@ private void cancel()
             close();
         }
 
+        private void CheckEscape(object sender, ButtonReleasedEventArgs args) {
+            if (DoingKeyBinding) return;
+            if (args.Button.ToString() == "Escape") {
+                Mod.instance.Helper.Events.Input.ButtonReleased -= CheckEscape;
+                cancel();
+                }
+            }
```

Comment:
This handles the event of user pressing Escape while not doing key binding. It is hardcoded to act similarly to pressing the "Cancel" button.